compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeNarrower.java

```diff
@@ -110,6 +111,30 @@ public SymbolEnv evaluateTruth(BLangExpression expr, BLangNode targetNode, Symbo
         return targetEnv;
     }
 
+    public SymbolEnv evaluateTruth(BLangExpression expr, BType typeToRemove, BLangNode targetNode, SymbolEnv env) {
```

Comment:
Shall we  add a doc comment explaning what this function do.

Reply:
Fixed